leaderboard/src/components/elements/Accordion/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ReactNode } from 'react';
+
+interface AccordionProps {
+  title: string | ReactNode;
+  content: string | ReactNode;
+}
+
+const Accordion = ({ title, content }: AccordionProps) => {
+  return (
+    <div className="collapse">
+      <input type="checkbox" />
+      <div className="collapse-title bg-white mb-2 text-neutral font-medium border border-[#E0E0E0] rounded-md">
+        {title}
+      </div>
+      <div className="collapse-content px-0">
+        <div className="p-4 bg-white border border-[#E0E0E0] rounded-md">
```

Comment:
If we are moving with a tailwind for styles lets also make it a theme-specific for discuss-board.